app/Jobs/EsDocument.php

```diff
@@ -66,5 +68,18 @@ public function handle(): void
         $keyName = $model->getKeyName();
         $model->setAttribute($keyName, $id);
         $model->esDeleteDocument();
+        static::incrementStat('delete', get_class($model));
```

Comment:
why one up there uses `$this->modelMeta(...)` and this one uses `get_class(...)` and I think they are all really just `$class`? or maybe don't bother passing the class name at all if the incrementStat function is an instance method?